scss/elements/forms/_checkbox.scss

```diff
@@ -0,0 +1,155 @@
+[type=checkbox] {
+  @warn "This selector is deprecated! use .checkbox-white or .checkbox-blue";
+  background-color: $white;
+  border: 0;
+  height: $space-3;
+  line-height: normal;
+  position: relative;
+  width: $space-3;
+
+  &::after {
+    background-color: $white;
+    border: 1px solid $grey-15;
+    border-radius: 3px;
+    color: $grey-50;
+    content: '';
+    cursor: pointer;
+    display: block;
+    height: $space-3;
+    position: absolute;
+    width: $space-3;
+    z-index: 1;
+  }
+
+  &:disabled::after {
+    background-color: $grey-10;
+    border-color: $grey-10;
+    border-radius: 3px;
+    content: '';
+    cursor: default;
+  }
+
+  &:disabled:hover::after {
+    border-color: $grey-10;
+  }
+
+  &[readonly] {
+    background: none;
+
+    &::after {
+      background: none;
+      border: 0;
+      cursor: default;
+    }
+  }
+
+  &:hover::after {
+    border-color: $grey-50;
+  }
+
+  &:checked {
+    @include icon(check);
+
+    &::before {
+      color: $grey-50;
+      font-size: $h6;
+      left: 3px;
+      position: absolute;
+      top: 4px;
+      z-index: 2;
+    }
+  }
+
+  &:disabled:checked::after {
+    border: 0;
+    cursor: default;
+    margin-left: 1px;
+    margin-top: 1px;
+  }
+}
+
+/* ------- WHITE CHECKBOX, GREY TICK ------- */
+.checkbox-white[type=checkbox] {
```

Comment:
the `[type=checkbox]` part of this can also be removed when the deprecated styles are removed, right?